packages/as-transformer/src/transformer.ts

```diff
@@ -23,6 +33,11 @@ export class Transformer extends TransformVisitor {
     return super.visitCallExpression(node);
   }
 
+  /**
+   * Visits the sources and replaces the {@link CallExpression} with the new expression.
```

Comment:
@gregLibert not sure about this method, is it the method called after AST generation used to apply customized AST operations on targeted sources before compile time ?

Reply:
it's one of the three transformer hooks as defined [here](https://www.assemblyscript.org/compiler.html#transforms). It is the first one, called just after parsing the code